app/services/credit_notes/create_service.rb

```diff
@@ -23,6 +23,7 @@ def call
 
       ActiveRecord::Base.transaction do
         result.credit_note = CreditNote.create!(
+          organization_id: invoice.organization_id,
```

Comment:
There is also the EstimateService where a credit note is created.

Reply:
yep, thanks. I have missed it as it calls `CreditNote.new` instead of `.create`.